dev-support/Jenkinsfile_GitHub

```diff
@@ -85,17 +85,7 @@ pipeline {
                         mvn --offline --version  || true
                         echo "getting machine specs, find in ${BUILD_URL}/artifact/patchprocess/machine/"
```

Comment:
This path is incorrect, mind removing this line? It is logged inside the script [1]. Sample output [2]

[1] https://github.com/apache/hbase/blob/636fa2c6b02d67427c2c33f71b8e9977e064debb/dev-support/gather_machine_environment.sh#L41
[2] https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-883/1/console

Reply:
Thank you!
I removed that line, hope it looks good now.